datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -443,7 +443,15 @@ def log_typos_in_options(self, user_config, models_config, level):
         models_config = models_config or {}
         typos = set()  # type: Set[str]
 
-        known_options = [k for k, _ in models_config]  # type: List[str]
+        known_options = set([k for k, _ in models_config])  # type: Set[str]
+
+        if not PY2:
+            from pydantic import BaseModel
```

Comment:
What do you think about importing this here instead? https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/base.py#L69

Reply:
That's what I did at the beginning but I will also need this if here anyway

Reply:
In fact you're right, I moved the import to the if statement you mentioned. 

Also, I need the if statement here because mypy complains otherwise because we are running it using `--py2`, so it's not defined if I do not add it.

Should be good now, let me know what you think!

Reply:
Oh interesting, I see the issue. I think this is fine then!